lib/qtermwidget.h

```diff
@@ -41,6 +41,21 @@ class QTermWidget : public QWidget {
         ScrollBarRight=2
     };
 
+    enum KeyboardCursorShape {
```

Comment:
Because `qtermwidget.h` is the header exposed to the app, and the app should use the enum.